Path.json

```diff
@@ -8978,6 +8978,88 @@
 				}
 			]
 		},
+		{
+			"electrified": true,
+			"group": 2,
+			"twistingFactor": 0.1,
+			"maxSpeed": 320,
+			"name": "LGV Est-Européenne",
+			"neededEquipments": [
+				"FR",
+				"ETCS"
```

Comment:
Statt ETCS neues Equipement TVM hinzufügen: BR 407 hat kein ETCS und könnte sonst dort nicht fahren…

Reply:
Dass die 407er kein ETCS haben, ist mir noch gar nicht aufgefallen 😅
Wäre eigentlich in echt ein Fahrzeug mit ETCS aber ohne TVM in der Lage, auf der Strecke zu fahren?

Reply:
Ja, das sollen die ICE, sobald sie nächstes Jahr nur noch ETCS, aber keine TVM mehr haben. Stand jetzt gibt es aber kein Fz ohne TVM, das auf französischen LGV fährt…

Reply:
Moment mal, derzeit steht das in der `Train.json` so drin 🤔

Reply:
Stimmt, ich hatte das damals so eingetragen, weil es ja keinen 403 gibt, der auf der VDE8 verkehren könnte. Naja, dann bleiben wir in der Zukunft, lassen dem 407 sein ETCS und als Streckenanforderung ebenfalls das ETCS.

Reply:
403 brauchen wir sagst du?

*403 022 umwurstgeräusche*

Reply:
> 403 brauchen wir sagst du?
> 
> _403 022 umwurstgeräusche_

Irgendwann werden wir auf jeden Fall noch 406 für den Eurotunnel brauchen :D

Reply:
Aber in der Praxis gibt es ohnehin keine LGV-fähigen Fahrzeuge mit ETCS, aber ohne TVM, oder?

Reply:
> Aber in der Praxis gibt es ohnehin keine LGV-fähigen Fahrzeuge mit ETCS, aber ohne TVM, oder?

D.h., wie sieht es mit dem ETR 1000 aus? 🤔

Reply:
403/406 Wurste ich mal eben, moment.
Aber im ET fahren eher 407 bzw lange Velaros :D

Reply:
DIE ETR 1000 haben TVM, weil auf der LGV Sud-Est nur TVM installiert ist.

Reply:
> > 403 brauchen wir sagst du?
> > _403 022 umwurstgeräusche_
> 
> Irgendwann werden wir auf jeden Fall noch 406 für den Eurotunnel brauchen :D

Wie soll 406 nach London, wenn er noch nichtmal mehr nach Frankreich kann?

Reply:
Mit ner Diesellok ziehen 😂

Reply:
> > > 403 brauchen wir sagst du?
> > > _403 022 umwurstgeräusche_
> > 
> > 
> > Irgendwann werden wir auf jeden Fall noch 406 für den Eurotunnel brauchen :D
> 
> Wie soll 406 nach London, wenn er noch nichtmal mehr nach Frankreich kann?

Ach, der hat auch die Zulassung richtig verloren? 😂